docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -453,72 +453,35 @@ _deviceContext->Draw(3, 0);
 ```
 
 And finally, tell the GPU to draw 3 vertices, this will invoke the vertex shader 3 times, and it will successfully process the 3 vertices we put in our vertex buffer.
-You should now be able to run this and see your first triangle!
-
-!!! error
-    Fix this graph madness
-```mermaid
-stateDiagram-v2
-[*] --> [Input Assembly Stage]
-Still --> [*]
-Still --> Moving
-Moving --> Still
-Moving --> Crash
-Crash --> [*]
-```
 
+Finally let's review all the commands we issue in `Render`
 
-```
-    [*] --> IA
-    IA[Input Assembler Stage] --> VS[Vertex Shader Stage]
-    VS -.-> HS[Hull Shader Stage]
-    HS[Hull Shader Stage] -.-> TS[Tesselation Shader Stage]
-    TS -.-> DS[Domain Shader Stage]
-    DS -.-> GS[Geometry Shader Stage]
-    GS --> RS[Rasterizer Stage]
-    GS --> SO[Stream Output]
-    RS --> PS[Pixel Shader Stage]
-    PS --> OM[Output Merger Stage]
-
-    MR[Buffer, Texture, Constant Buffer] --> IA
-    MR-->VS
-    MR-->HS
-    MR-->DS
-    MR-->GS
-
-    MR-->PS
-    PS-->MR
-    MR-->OM
-    OM-->MR
-
-    SO-->MR
-    SO --> [*]
+```cpp
+_deviceContext->IASetInputLayout(_vertexLayout.Get());
+_deviceContext->IASetVertexBuffers(
+    0,
+    1,
+    _triangleVertices.GetAddressOf(),
+    &vertexStride,
+    &vertexOffset);
+_deviceContext->IASetPrimitiveTopology(D3D11_PRIMITIVE_TOPOLOGY::D3D11_PRIMITIVE_TOPOLOGY_TRIANGLELIST);
+_deviceContext->VSSetShader(
+    _vertexShader.Get(),
+    nullptr,
+    0);
+_deviceContext->RSSetViewports(
+    1,
+    &viewport);
+_deviceContext->PSSetShader(
+    _pixelShader.Get(),
+    nullptr,
+    0);
+_deviceContext->OMSetRenderTargets(
+    1,
+    _renderTarget.GetAddressOf(),
+    nullptr);
 ```
 
-```mermaid
-graph TD
-  IA[Input Assembler Stage] --> VS[Vertex Shader Stage];
-  VS -.-> HS[Hull Shader Stage];
-  HS[Hull Shader Stage] -.-> TS[Tesselation Shader Stage];
-  TS -.-> DS[Domain Shader Stage];
-  DS -.-> GS[Geometry Shader Stage];
-  GS --> RS[Rasterizer Stage];
-  GS --> SO[Stream Output];
-  RS --> PS[Pixel Shader Stage];
-  PS --> OM[Output Merger Stage];
-
-  MR[Buffer, Texture, Constant Buffer] --> IA;
-  MR-->VS;
-  MR-->HS;
-  MR-->DS;
-  MR-->GS;
-
-  MR-->PS;
-  PS-->MR;
-  MR-->OM;
-  OM-->MR;
-
-  SO-->MR;
-```
+As you can see, we go through the pipeline in an orderly fashion, and although we don't use all the stages, we can see the top-to-bottom execution of the stages, `IA -> VS -> RS -> PS -> OM`.
```

Comment:
hehe, almost. Instead of writing IA VS etc.. either we keep a legend/sentence somewhere that we will be referring to VertexShader stage as `VS` and input assembly stage as `IA` etc... or writing it out, as the picture at the beginning says